week2/index.js

```diff
@@ -39,8 +39,8 @@ function listTodos() {
         return console.log('Error: Something went wrong', error);
       }
     }
-
-    var todos = splitStringByNewline(data);
+ 
+    let todos = splitStringByNewline(data);
```

Comment:
Good use of ES6 syntax!
Did you consider `const todos = ....` since it doesn't change?